src/app/about/page.tsx

```diff
@@ -1,5 +1,92 @@
+import Image from "next/image";
+
+
 const About = () => {
-  return <div>Hello</div>;
-};
+  return (
+    <main className="min-h-screen">
+      {/* Navbar */}
+      <nav className="bg-white shadow-md">
+  <div className="flex flex-col items-center md:flex-row md:justify-between md:items-center p-6 bg-white">
+    {/* Title */}
+    <h1 className="text-3xl font-bold text-red-600 mb-4 md:mb-0">
+      Martha Pollack's Portfolio
+    </h1>
+    
+    {/* Line below the title on mobile (hidden on desktop) */}
+    <div className="w-full border-t-2 border-red-300 mt-2 mb-4 md:hidden"></div>
+
+    {/* Navigation Links */}
+    <ul className="flex flex-col w-full items-center space-y-1 md:flex-row md:space-y-0 md:space-x-6 md:w-auto">
+      {/* ABOUT Link with line above on desktop */}
+      <li className="relative w-full md:w-auto">
+        <a 
+          href="#about" 
+          className="block text-center text-red-600 font-bold md:text-left md:inline-block md:border-t-4 border-red-600 py-2 w-full md:w-auto"
+        >
+          ABOUT
+        </a>
+      </li>
+
+      {/* Other Links */}
+      <li className="w-full md:w-auto">
+        <a 
+          href="#achievements" 
+          className="block text-center text-red-300 hover:text-red-600 py-2 w-full md:w-auto"
+        >
+          ACHIEVEMENTS
+        </a>
+      </li>
+      <li className="w-full md:w-auto">
+        <a 
+          href="#fun-facts" 
+          className="block text-center text-red-300 hover:text-red-600 py-2 w-full md:w-auto"
+        >
+          FUN FACTS
+        </a>
+      </li>
+      <li className="w-full md:w-auto">
+        <a 
+          href="#contact" 
+          className="block text-center text-red-300 hover:text-red-600 py-2 w-full md:w-auto"
+        >
+          CONTACT
+        </a>
+      </li>
+    </ul>
+  </div>
+</nav>
+
+
+
+      {/* Main Content Section Only */}
+      <div className="flex flex-col md:flex-row items-center justify-center max-w-6xl mx-auto mt-12 p-8">
+        {/* Avatar Section */}
+        <div className="flex-shrink-0 mb-6 md:mb-0 md:mr-12">
+          <Image
+            src="/image.png" 
+            alt="Martha Pollack"
+            className="rectangle"
```

Comment:
"rectangle" is not the best name for an image class, consider changing it to "profile-pic-container" or something like that